perfkitbenchmarker/benchmark_spec.py

```diff
@@ -182,6 +195,21 @@ def ConstructVirtualMachines(self):
         os_type = self._GetOsTypeForGroup(group_name)
         cloud = self._GetCloudForGroup(group_name)
         providers.LoadProvider(cloud.lower())
+        ProviderInfoClass = provider_info.GetProviderInfoClass(cloud.lower())
+
+        benchmarkOkay = True
```

Comment:
nit: to stay consistent with the rest of the code base please use underscores (benchmark_okay) rather than camel case for local variables